sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/entity_resolution/Commons.scala

```diff
@@ -71,16 +75,19 @@ class EntityResolution_RDFData_CountVectorizer(spark: SparkSession, triplesRDD1:
    * where subject is the key and the group of paired predicate:object forms the value
    * ex:- (Budapest_City, areaCode:1 , country:Hungary)
    */
-  def get_entity_profiles(triplesRDD: RDD[Triple]): RDD[(String, String)] = {
+  def getEntityProfiles(sourceData: RDD[Triple]): RDD[(String, String)] = {
 
     //predicates to be filtered out from triples
```

Comment:
Not, sure if you have run `mvn scalastyle:check` before you pushed the changes, but what I can see Scala Style Check will complain about this one (see here: http://www.scalastyle.org/rules-dev.html#org_scalastyle_scalariform_SpaceAfterCommentStartChecker ).

Reply:
Do the same for the rest!

Reply:
Done.